mm2src/coins/utxo/utxo_common.rs

```diff
@@ -800,13 +799,10 @@ impl<'a, T: AsRef<UtxoCoinFields> + UtxoTxGenerationOps> UtxoTxBuilder<'a, T> {
 /// returns transaction and data as is if the coin is not KMD
 pub async fn calc_interest_if_required<T: UtxoCommonOps>(
     coin: &T,
-    mut unsigned: TransactionInputSigner,
-    mut data: AdditionalTxData,
-    my_script_pub: Bytes,
-    dust: u64,
-) -> UtxoRpcResult<(TransactionInputSigner, AdditionalTxData)> {
-    if coin.as_ref().conf.ticker != "KMD" {
-        return Ok((unsigned, data));
+    unsigned: &mut TransactionInputSigner,
+) -> UtxoRpcResult<u64> {
+    if !coin.is_kmd() {
+        return Ok(0);
     }
     unsigned.lock_time = coin.get_current_mtp().await?;
     let mut interest = 0;
```

Comment:
we should replace `get_verbose_transaction` with `get_verbose_transactions_from_cache_or_rpc` since we run this inside the tx solver loop.

sorry can't annotate the specific line (L#814 = 6 lines below) cuz it's not changed.

Reply:
fixed in 7ca03f4078336d05fb5e014ba561ec3b06ea0ae4 thanks